projects/saturn/src/main/java/io/fairspace/saturn/services/views/ViewService.java

```diff
@@ -175,23 +175,27 @@ public List<ViewDTO> getViews() {
         return viewsConfig.views.stream()
                 .map(v -> {
                     var columns = new ArrayList<ColumnDTO>();
-                    columns.add(new ColumnDTO(v.name, v.itemName == null ? v.name : v.itemName, ColumnType.Identifier));
+
+                    // The entity labal is the first column displayed, if you want a column before this label, assign a negative displayIndex value in views.yaml
+                    int entityLabelIndex = 0;
```

Comment:
Sorry for confusion, I did not mean that, was talking not about method param name, but the value we pass there. What I mean is that when I was reading the code I did not get why we put 0 there. But if we put a const with proper naming, it would be clear. Constant on class level like this:

`private final static int ENTITY_LABEL_INDEX = 0` (or public, so could be used in tests for asserts if we do the check there.

It also can stay be a const on method level, but in this case I would suggest to add 'final'.

And here is the [link](https://rules.sonarsource.com/java/RSPEC-109/) describing the magic numbers, Sonar will fail once we switch it on

Reply:
if we had a unit test on the getViews method, it also could be explain a reader what is expected and why, but there is no one :( Let's skip it for now, I think we will get back to it once code coverage is switched on